Makefile

```diff
@@ -423,7 +433,12 @@ export LANG := C
 
 else # TARGET_N64
 
+ifneq ($(TARGET_OSX),1)
 AS := as
+else
+AS := i686-w64-mingw32-as
+endif
+
 ifneq ($(TARGET_WEB),1)
   CC := gcc
   CXX := g++
```

Comment:
I tried clang and it seems to compile just fine for me
perhaps explicitly replacing these with clang and clang++ for MacOS is better instead of relying on the symlinks

Reply:
@Jan200101 Your comment is linked to the section that defines `CC` & `CXX` for everything but `TARGET_WEB`, which I haven't made any changes to. Did you mean to comment on a different part of this `Makefile`?